docs/data/sql_functions.yml

```diff
@@ -241,9 +241,9 @@ arithmetic:
   - sql: UNHEX(expr)
     table: expr.unhex()
     description: |
-      Converts hexadecimal string expr to BINARY.
+      Converts hexadecimal string expr to BINARY. If the length of expr is odd, the first character is discarded and the result is left padded with a null byte. 
       
-      If the length of expr is odd, the first character is discarded and the result is left padded with a null byte. 
+      E.g., SELECT UNHEX('466C696E6B') or $('466C696E6B').unhex() returns 466C696E6B as BINARY.
```

Comment:
usually we should update both english and chinese docs
can you also add examples there (could be in english for now however it is better to keep them sync)

Reply:
Sure, I planned to create a ticket for it, but adding the english version first sounds good and then creating a ticket for translation sounds good.

Reply:
Done. Thanks for taking a look @snuyanzin